pywhyllm/suggesters/simple_identification_suggester.py

```diff
@@ -1,7 +1,7 @@
 import guidance
 from guidance import system, user, assistant, gen
 import re
-
+from inspect import cleandoc
```

Comment:
the other files use "import inspect" and then call inspect.cleandoc, while this one imports cleandoc directly.  It would be better to be consistent in usage

Reply:
good idea. I used "from inspect import cleandoc" everywhere.